src/main/java/com/blackducksoftware/integration/hub/rest/RestConnection.java

```diff
@@ -262,8 +262,8 @@ private Response handleExecuteClientCall(final Request request, final int retryC
                     return handleExecuteClientCall(request, retryCount + 1);
                 } else {
                     throw new HubIntegrationException(
-                            "There was a problem trying to " + request.method() + " this item : " + request.url().uri().toString() + ". Error : "
-                                    + response.message());
+                            String.format("There was a problem trying to %1$s this item: %2$s. Error: %3$s %4$s",
```

Comment:
The %1$s should be changed to %s to be consistent everywhere, unless you are specifically using the same segment multiple times or out of order